server/config/config.js

```diff
@@ -10,17 +10,17 @@ module.exports = {
     "dialect": "mysql"
   },
   "test": {
-    "username": "root",
-    "password": "password",
-    "database": "tfgdb_test",
-    "host": "localhost",
+    "username": "b976cfa96691f2",
+    "password": "e1b56bb4",
+    "database": "heroku_cf5c4e9e2e2106c",
+    "host": "eu-cdbr-west-02.cleardb.net",
     "dialect": "mysql"
   },
   "production": {
-    "username": "root",
-    "password": null,
-    "database": "database_production",
-    "host": "127.0.0.1",
+    "username": "b670e3c59c4b4d",
```

Comment:
Codacy has a fix for the issue: [Replace `"username"` with `username`](https://app.codacy.com/gh/manrubben/TFG/pullRequest?prid=9595920)
```suggestion
    username: "b670e3c59c4b4d",
```